### ISSUE TITLE
SRGAN implementations in another frameworks

### ISSUE THREAD

**Issue by crcrpar** [crcrpar/noituloser_repus_reniahc#2]
[TensorLayer](https://github.com/tensorlayer/srgan) uses DIV2K dataset.
In summary,
- batch size: 16
- initial learning rate is 1e-4
- epoch: 100 for only generator training
- epoch: 2000 for adversarial training. Learning rate is divided by 10 after 1000 epochs.

Details: https://github.com/tensorlayer/srgan/blob/master/config.py
